src/cli/cli_joiner.cpp

```diff
@@ -86,6 +127,20 @@ template <> otError Joiner::Process<Cmd("id")>(Arg aArgs[])
     return OT_ERROR_NONE;
 }
 
+/**
+ * @cli joiner start
+ * @code
+ * joiner start J01NM3
+ * Done
+ * @endcode
+ * @cparam joiner start @ca{joining-device-credential} [@ca{provisioning-url}]
+ * *   `joining-device-credential`: %Joiner Passphrase. Must be a string of all uppercase alphanumeric
```

Comment:
What is the % used for here?  I see it prefixing Joiner in multiple places.  It's not mentioned in the contribute page

Reply:
I updated the Contributing page and added some instructions for the `%`. It's used to tell Doxygen to NOT link to a word...it kept thinking that Joiner was a class name...